README.md

```diff
@@ -518,6 +521,23 @@ Use `scripts/make_template.py` to dump 1D/2D histogram from `.coffea` to `TH1D/T
 </p>
 </details>
 
+## Submit standalone condor jobs
+
+You have the option to run the framework through "standalone condor jobs", bypassing the native coffea-supported job submission system. Within each job, a standalone script will execute the following on the worker node:
+
+ - Set up a minimal required Python environment.
+ - Retrieve the BTVNanoCommissioning repository, either from a git link or transferred locally.
+ - Launch the `python runner.py ...` command to execute the coffea framework in the iterative executor mode.
+ 
+This utility is currently adapted for the lxplus and cmsconnect condor systems. To generate jobs for launching, replace `python runner.py` with `python condor/submitter.py`, append the existing arguments, and add the following arguments in addition:
+
+ - `--jobName`: Specify the desired condor job name. A dedicated folder will be generated, including all submission-related files.
+ - `--outputXrootdDir`: Indicate the XRootD directory's path (starting with `root://`) where the produced .coffea (and .root) files from each worker node will be transferred to.
+ - `--condorFileSize`: Define the number of files to process per condor job (default is 50). The input file list will be divided based on this count.
```

Comment:
Do we have to give some tips on optimizing this number depending on which machine you are using for submission? e.g. lxplus condor is quite limited, so I guess this number is already optimized for lxplus?

Reply:
I agree with @colizz  for the comments on the limitation. The bare condor submission should be fine, just the usual waiting lines. I would say 2GB (optimized previously) is a nice size easier to get jobs running. The nice thing is the jobs are not limited with the scheduler(namely 100 from `dask` setting)